tycho-common/src/dto.rs

```diff
@@ -1428,9 +1428,16 @@ pub struct TracedEntryPointRequestResponse {
     pub pagination: PaginationResponse,
 }
 
+#[derive(Serialize, Deserialize, Debug, Default, PartialEq, ToSchema, Eq, Clone)]
 pub struct AddEntrypointRequestBody {
-    pub entrypoints_with_tracing_data: Vec<(String, Vec<EntryPointWithTracingParams>)>,
+    #[serde(default)]
     pub chain: Chain,
+    /// The map of entry point ids to their tracing params to insert and optionally a component id
+    /// used for debugging only.
+    // TODO fix this warning
+    #[allow(clippy::type_complexity)]
+    pub entry_points_with_tracing_data:
+        Vec<(String, Vec<(EntryPointWithTracingParams, Option<String>)>)>,
```

Comment:
I can't see Jira right now - was the first param meant to be the component id and not the entry point ids? That would make a lot more sense haha, since the entry point id right now is kindof useless. Then I could remove the innermost Option<String> which should represent the component id